Mathlib/Geometry/Manifold/VectorBundle/FiberwiseLinear.lean

```diff
@@ -249,7 +249,7 @@ def smoothFiberwiseLinear : StructureGroupoid (B × F) where
       (h2φ : SmoothOn IB 𝓘(𝕜, F →L[𝕜] F) (fun x => (φ x).symm : B → F →L[𝕜] F) U),
         {e | e.EqOnSource (FiberwiseLinear.partialHomeomorph φ hU hφ.continuousOn h2φ.continuousOn)}
   trans' := by
-    simp only [mem_aux]
+    simp only [mem_iUnion]
```

Comment:
`mem_aux` is a private lemma right before this one which is tagged with a porting note
"/- Porting note: `simp only [mem_iUnion]` fails in the next definition. This aux lemma is a
workaround. -/"
This reverts the workaround in three places.